app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts
+  has_many :comments
+  has_many :likes
```

Comment:
- [x] Good job creating the association with `has_many` on the user model side 👏 .  However, you should remember that the  `author_id` column in posts/likes/comments table should be the `foreign_key` for the users table. To ensure the link between the posts/likes/comments table are link to the user table, kindly fix that in the `user` model 😄 

```ruby
has_many :posts, foreign_key: 'author_id'
has_many :comments, foreign_key: 'author_id'
has_many :likes, foreign_key: 'author_id'
```